src/routes/Addons/Addons.js

```diff
@@ -182,6 +182,7 @@ const Addons = ({ urlParams, queryParams }) => {
                             className={styles['addon-url-input']}
                             type={'text'}
                             placeholder={'Paste addon URL'}
+                            autoFocus
```

Comment:
its a minor thing but could you change that to `autoFocus={true}`

Reply:
Sure !